### PR TITLE
Feature/aruha 115

### PR DESCRIPTION
**Pull request by jkliff**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/169" target="_blank">ReviewNinja

File: api/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -367,11 +367,13 @@ paths:
...
-        will also be removed.
-        It's important to note that topic deletion in Kafka can take some time, so before it is completelly removed
-        it will not be possible to create an `EventType` with the same name as removed one had (you will get 409
-        response code from creation endpoint)
+        Deletes an `EventType` identified by its name. This operation happens asyncrhonously in the
+        background and might take up to the retention time for the underlying topic to succeed.
+
+        Attempting to immediatelly recreate an EventType with the same name might fail (409 Conflict).
+
+        This operation is supposed ot be used by Nakadi operators and rights for its use might not granted
```
Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: /ot/to/

Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: /not granted/not be granted/

File: api/nakadi-event-bus-api.yaml [hunk 2]
```diff
@@ -1126,18 +1134,6 @@ definitions:
             `metadata` and any additionally defined properties passed in the `EventTypeSchema` directly on
             top level of the Event. If name conflicts arise, creation of this EventType will be rejected.
             Default validation_strategy for this `EventType` is `[{name: 'schema-validation'}]`.
-      effective_schema:
```
Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: There are some references to this field spread in this file. I think we should search for then and remove as well.

File: api/nakadi-event-bus-api.yaml [hunk 3]
```diff
@@ -399,7 +401,11 @@ paths:
         '404':
           description: EventType not found.
           schema:
-              $ref: '#/definitions/Problem'
+            $ref: '#/definitions/Problem'
+        '409':
```
Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: In which case 409 is possible here?

File: api/nakadi-event-bus-api.yaml (hunk 1 above)
Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: The errors above are still there.

**Comment by rcillo** [comment]
:shipit:

**Pull request merged**